src/extension/devtools/devtools.ts

```diff
@@ -161,26 +157,17 @@ async function createDevtoolsPanel() {
         panelWindow.send({ type: "devtoolsStateChanged", state: value });
       });
 
+      clientPort.forward("explorerResponse", panelWindow);
+      panelWindow.forward("explorerRequest", clientPort);
+      panelWindow.forward("explorerSubscriptionTermination", clientPort);
+
       connectedToPanel = true;
     }
 
     if (devtoolsMachine.state.value === "initialized") {
       clientPort.send({ type: "connectToClient" });
       startConnectTimeout();
     }
-
-    removeExplorerForward = clientPort.forward("explorerResponse", panelWindow);
```

Comment:
This was run every time the user changed tabs in the devtools, then navigated back to Apollo devtools. Since this just sets up forwarding between the two "actors" so that the message passing can communicate between the two, no reason why we needed to cut off these messages when the user navigates away. Now we'll just set them up the first time the panel is created and call it good.